### PR TITLE
Update API url

### PR DESCRIPTION
**Pull request by dominik-zeglen**
Self explanatory

### CODE REVIEW

**Comment by krzysztofwolski** [comment]
@dominik-zeglen can we also update GH Actions? Test envs for this release were pointing to .rocks domain

**Review by jwm0 — APPROVED** [review]
✨

File: .github/workflows/test-env-deploy.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ jobs:
         id: set-domain
         # Set test instance domain based on branch name slug
         run: |
-          echo "::set-output name=domain::${{ env.GITHUB_HEAD_REF_SLUG_URL }}.dashboard.saleor.rocks"
```
Comment (cmiacz) [inline .github/workflows/test-env-deploy.yml:76]: It should not be modified. Test environments are hosted outside cloud, under saleor.rocks domain.

File: .github/workflows/test-env-cleanup.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
         id: set-domain
         # Set test instance domain based on branch name slug
         run: |
-          echo "::set-output name=domain::${{ env.GITHUB_HEAD_REF_SLUG_URL }}.dashboard.saleor.rocks"
```
Comment (cmiacz) [inline .github/workflows/test-env-cleanup.yml:17]: It should not be modified. Test environments are hosted outside cloud, under saleor.rocks domain.

**Review by cmiacz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**